Library/Formula/gnuradio.rb

```diff
@@ -0,0 +1,195 @@
+    sha256 "0703c1ea5a6af0bb6d0cec24708301334949d56ebc7f95c64028d9c66f9d8d5d"
+  end
+
+  resource "babel" do
+    url "https://pypi.python.org/packages/source/B/Babel/Babel-1.3.tar.gz"
+    sha256 "9f02d0357184de1f093c10012b52e7454a1008be6a5c185ab7a3307aceb1d12e"
+  end
+
+  resource "snowballstemmer" do
+    url "https://pypi.python.org/packages/source/s/snowballstemmer/snowballstemmer-1.2.0.tar.gz"
+    sha256 "6d54f350e7a0e48903a4e3b6b2cabd1b43e23765fbc975065402893692954191"
+  end
+
+  resource "six" do
+    url "https://pypi.python.org/packages/source/s/six/six-1.9.0.tar.gz"
+    sha256 "e24052411fc4fbd1f672635537c3fc2330d9481b18c0317695b46259512c91d5"
+  end
+
+  resource "pytz" do
+    url "https://pypi.python.org/packages/source/p/pytz/pytz-2015.2.tar.bz2"
+    sha256 "3e15b416c9a2039c1a51208b2cd3bb4ffd796cd19e601b1d2657afcb77c3dc90"
+  end
+  # sphinx ends here
+
+  resource "cppzmq" do
+    url "https://github.com/zeromq/cppzmq/raw/34c8e4395c94d34a89bbeaaf2b8f9c94a8293c84/zmq.hpp"
+    sha256 "6cdd9d920f4a0f9f3a3257541321bef8369d9735c88b84ba8ae0e461f53e476c"
+  end
+
+  def install
+    build_python = (build.with? "python") || (build.with? "wx") || (build.with? "qt")
+
+    ENV["CHEETAH_INSTALL_WITHOUT_SETUPTOOLS"] = "1"
+    ENV.prepend_create_path "PYTHONPATH", libexec+"vendor/lib/python2.7/site-packages"
+
+    res = %w[Markdown Cheetah lxml]
+    res += %w[sphinx sphinx_rtd_theme alabaster babel docutils pygments
+              jinja2 markupsafe snowballstemmer six pytz] if build.with? "documentation"
+    res.each do |r|
+      resource(r).stage do
+        system "python", *Language::Python.setup_install_args(libexec/"vendor")
+      end
+    end
+
+    resource("cppzmq").stage "#{include}" if build.with? "zeromq"
+
+    ENV.prepend_path "PATH", (libexec+"vendor/bin")
+
+    args = std_cmake_args
+
+    if build.universal?
+      ENV.universal_binary
+      args << "-DCMAKE_OSX_ARCHITECTURES=#{Hardware::CPU.universal_archs.as_cmake_arch_flags}"
+    end
+
+    args << "-DENABLE_DEFAULT=OFF"
+    enabled_componenets = %w[gr-analog gr-fft volk gr-filter gnuradio-runtime
+                             gr-blocks testing gr-pager gr-noaa gr-channels
+                             gr-audio gr-fcd gr-vocoder gr-fec gr-digital
+                             gr-dtv gr-atsc gr-trellis]
+    enabled_componenets << "gr-qtgui" if build.with? "qt"
+    enabled_componenets << "gr-video-sdl" if build.with? "sdl"
+    enabled_componenets << "gr-wavelet" if build.with? "wavelet"
+    enabled_componenets << "gr-uhd" if build.with? "uhd"
+    enabled_componenets << "gr-wxgui" if build.with? "wx"
+    enabled_componenets << "gr-zeromq" if build.with? "zeromq"
+    enabled_componenets += %w[python gr-ctrlport grc gr-utils] if build_python
+    enabled_componenets += %w[doxygen sphinx] if build.with? "documentation"
+    enabled_componenets.each do |c|
+      args << "-DENABLE_#{c.upcase.split("-").join("_")}=ON"
```

Comment:
You can replace hyphens with underscores above and remove this `.split("-").join("_")`.